zebedee-reader/Dockerfile.local

```diff
@@ -0,0 +1,25 @@
+#### Local Dockerfile ####
+# This file is intended to be used by dp-compose for local deployments of zebedee reader.
+# The entrypoint is configurable via some env vars that must be provided by the docker compose definition
+##########################
+
+FROM openjdk:8-jre
+
+# Add the repo source
+WORKDIR /usr/src
+ADD ./target/dependency /usr/src/target/dependency
+ADD ./target/classes /usr/src/target/classes
+
+# Temporary: expose Elasticsearch
```

Comment:
Why is this marked as temporary?

Reply:
I used the main docker file (cms) as a reference (https://github.com/ONSdigital/zebedee/blob/develop/Dockerfile) - and just replaced the things that needed to be replaced:
- Add reader dependency and classes folders (instead of CMS equivalents)
- Add env vars to configure some parameters

To be honest I don't know if this port needs to be exposed, but I did it so it's consistent with the cms DockerFile.

Reply:
I validated that this port is not being used in any manifest or stack. I will remove it.